src/test/java/Model/CarCheckTest.java

```diff
@@ -0,0 +1,44 @@
+package Model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class CarCheckTest {
+
+    private CarCheck carCheck = new CarCheck();
+
+    @Test
+    @DisplayName("조건에 어긋나는 이름을 입력해서 오류가 나는지 테스트")
+    void canCarCheckError() {
+        // given
+        String wrongInput = "abcde,abcdef,abcd,a,b";
+
+        // when
+
+        // then
+        assertThatThrownBy(() -> carCheck.carCheck(wrongInput))
+                .isInstanceOf(IllegalStateException.class);
```

Comment:
```suggestion
        // given
        String wrongInput = "abcde,abcdef,abcd,a,b";

        // when, then
        assertThatThrownBy(() -> carCheck.carCheck(wrongInput))
                .isInstanceOf(IllegalStateException.class);
```

비워두지 말고 위와 같이 해보는건 어떠신가요?

Reply:
when과 then이 동시에 해당하는 경우임을 인지 후 수정했습니다!